components/dialogs/AuthDialog.vue

```diff
@@ -388,8 +388,11 @@ export default {
       try {
         await apiCheckIsUser(this.getLocalWallet);
       } catch (err) {
-        // Assume it is 404
-        // Redirect user to register
+        if (!err.response || err.response.status !== 404) {
+          this.currentTab = 'signInError';
+          return;
```

Comment:
prefer `if (status === 404) ` so that later it can be easily swapped to a `switch` if needed?

Reply:
Updated